payjoin-cli/Cargo.toml

```diff
@@ -24,6 +24,8 @@ native-certs = ["reqwest/rustls-tls-native-roots"]
 _danger-local-https = ["rcgen", "reqwest/rustls-tls", "rustls", "hyper-rustls", "payjoin/_danger-local-https", "tokio-rustls"]
 v1 = ["payjoin/v1","hyper", "hyper-util", "http-body-util"]
 v2 = ["payjoin/v2", "payjoin/io"]
+hyper = ["dep:hyper"]
+http-body-util = ["dep:http-body-util"]
```

Comment:
It seems that you've added hyper as an optional dependency which is only enabled when the "hyper" feature is enabled, which it only is when using the `v1` feature. If you're building `payjoin-cli` with the default feature set that does not enable `v1` or the `hyper` feature you've defined here. Rather than making specific features for hyper and http-body-util, just enable them in the appropriate feature combination by including them in the `[...]` lists or making them default (by removing optional = true) if they are used with every feature combination